composer.json

```diff
@@ -1,22 +1,35 @@
 {
-	"name": "nzta/googlemap-leafletfield",
-	"description": "Provides a form field type allowing users to draw features on a map, the features are stored as geoJSON objects in a single database field",
-	"type": "silverstripe-module",
-	"homepage": "https://github.com/nzta/googlemap-leafletfield",
-	"keywords": ["googlemap", "leaflet", "map"],
-	"support": {
-		"issues": "https://github.com/nzta/googlemap-leafletfield/issues"
-	},
-	"authors": [
-		{
-			"name": "Robby",
-			"email": "channels@nzta.govt.nz"
-		}
-	],
-	"require": {
-		"silverstripe/framework": "~3.1"
-	},
-	"extra": {
-		"installer-name": "leafletfield"
-	}
+    "name": "nzta/googlemap-leafletfield",
+    "description": "Provides a form field type allowing users to draw features on a map, the features are stored as geoJSON objects in a single database field",
+    "type": "silverstripe-vendormodule",
+    "homepage": "https://github.com/nzta/googlemap-leafletfield",
```

Comment:
Would be good to add a license here of `BSD-3` as well as the actual license file itself